apps/meteor/app/apps/server/bridges/listeners.ts

```
@@ -0,0 +1,366 @@
+			.getConverters()
+			.get('rooms')
+			.convertAppRoom(result as IAppsRoom);
+	}
+
+	async livechatEvent(inte: keyof IListenerExecutor, payload: unknown[]): Promise<unknown> {
+		const [data] = payload;
+
+		switch (inte) {
+			case AppInterface.IPostLivechatAgentAssigned:
+			case AppInterface.IPostLivechatAgentUnassigned:
+				const agentData = data as LivechatAgentData;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)